test/test.ts

```diff
@@ -13,7 +13,7 @@ import { Agent, RequestOptions } from '../src';
 // versions we have to patch the internal `_http_agent` module instead
 // (see: https://github.com/nodejs/node/pull/25170).
 // @ts-ignore
-import httpAgent from '_http_agent';
+import * as httpAgent from '_http_agent';
```

Comment:
Note that the `// @ts-ignore` means we don't have to do the `require()` hack that was necessary for `https` below.